src/common/network/NetworkUtils.cpp

```diff
@@ -306,6 +306,19 @@ StatusOr<std::vector<HostAddr>> NetworkUtils::toHosts(const std::string& peersSt
     return hosts;
 }
 
+std::string NetworkUtils::toHosts(const std::vector<HostAddr>& hosts) {
```

Comment:
concatHosts may be a better func name to explain this?

Reply:
Thx. I prefer the `toHosts` function overload.

Reply:
`concatHosts` +1

Reply:
IMO, the **Concat** generally refers to joining the same kind of things and returning the original form. For example, joining multiple strings returns a string. There is a formal shift here(HostAddr **to** String, with the meaning of convert to), not a simple join or concat.

Reply:
toHosts +1